mobile/test/java/io/envoyproxy/envoymobile/engine/EnvoyConfigurationTest.kt

```diff
@@ -341,6 +347,33 @@ class EnvoyConfigurationTest {
     assertThat(resolvedTemplate).contains("initial_fetch_timeout: 5432s");
   }
 
+  @Test
+  fun `test adding RTDS and CDS`() {
+    JniLibrary.loadTestLibrary()
+    val envoyConfiguration = buildTestEnvoyConfiguration(
+      cdsResourcesLocator = "FAKE_CDS_LOCATOR", cdsTimeoutSeconds = 356, adsAddress = "FAKE_ADDRESS", adsPort = 0, enableCds = true
+    )
+
+    val resolvedTemplate = TestJni.createYaml(envoyConfiguration)
+
+    assertThat(resolvedTemplate).contains("FAKE_CDS_LOCATOR");
+    assertThat(resolvedTemplate).contains("FAKE_ADDRESS");
+    assertThat(resolvedTemplate).contains("initial_fetch_timeout: 356s");
```

Comment:
perhaps also a test case with no cdsResourcesLocator and timeout specified (and make sure they are the empty string and default timeout)